### PR TITLE
fix link url

### PR DESCRIPTION
**Pull request by moeryomenko**
add pre-commit hook for validate links.

the previous [pull request](https://github.com/rms-support-letter/rms-support-letter.github.io/pull/1771) made a mistake in the link.

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>

### CODE REVIEW

File: pre-commit [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```
Comment (gorshunovr) [inline pre-commit:1]: 1. Not everyone works on *nix system. Many users are on Windows or use GitHub UI.

**Comment by imachug** [comment]
I think it'd be easier for users if we ran the check in a GitHub Action. This would help many people who commit via Github UI.

**Comment by moeryomenko** [comment]
> I think it'd be easier for users if we ran the check in a GitHub Action. This would help many people who commit via Github UI.

Maybe in next pull request

File: pre-commit (hunk 1 above)
Comment (moeryomenko) [reply pre-commit]: Change README.md, mark this step for *nix users.

**Comment by imachug** [comment]
I'm just afraid that no one will bother to sign the letter if you require them to copy some potentially dangerous executables

**Comment by moeryomenko** [comment]
> if you require them to copy some potentially dangerous executables

Make optional

> some potentially dangerous executables

I disagree, the hook code is visible

**Comment by imachug** [comment]
Well, you know. If people don't care enough to double-check that the file format is valid, they surely won't bother to copy and check some scripts.

**Comment by moeryomenko** [comment]
> Well, you know. If people don't care enough to double-check that the file format is valid, they surely won't bother to copy and check some scripts.

Would agree, it's good point add validation

**Comment by imachug** [comment]
We now have a GitHub Action so closing.

**Pull request closed without merging**